source/troubleshooting.txt

```diff
@@ -173,3 +173,141 @@ HOMEDRIVE, HOMEPATH, or USERPROFILE are blank
 
 This error might appear if the {+atlas-cli+} can't access your home
 directory.
+
+.. _local-deploy-errors:
+
+Troubleshoot Local Atlas Deployment Issues
+------------------------------------------
+
+Error: qemu exited unexpectedly with exit code -1, stderr: qemu-system-x86_64: Error: HV_DENIED
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The {+atlas-cli+} uses `Podman <https://podman.io/>`__ and `QEMU 
+<https://www.qemu.org/>`__. We have identified this issue on MacOS x86 
+architectures. To resolve this isseu, upgrade to the latest version of 
+QEMU and restart Podman.
+
+Error: exit status 127
+~~~~~~~~~~~~~~~~~~~~~~
+
+This error might occur when you use the ``atlas deployments setup`` 
+command on Ubuntu Linux because of a `known issue 
+<https://bugs.launchpad.net/ubuntu/+source/libpod/+bug/2024394>`__ with 
+Podman. We don't support Ubuntu for local |service| deployments during 
+this Public Preview stage.
+
+MongoNetworkError: connect ECONNREFUSED
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This issue might occur after your machine goes into sleep mode or 
+restarts. When you try to connect to the local |service| deployment, 
+this error occurs.
+
+The {+atlas-cli+} uses `Podman <https://podman.io/>`__ and `QEMU 
+<https://www.qemu.org/>`__ to run dockerized instances of MongoDB. 
+Podman has a daemonless architecture. So, the local MongoDB instances 
+don't restart automatically.
+
+To fix this issue:
+
+1. List the Podman containers.
+
+   .. code-block:: sh
+
+      podman ps --all
+
+#. To resume the container, copy and paste the following command into 
+   your terminal, and replace ``{container-name}`` with the name of the 
+   container to start.
+
+   .. code-block:: sh
+
+      podman start {container-name}
+
+Issues on Windows
+~~~~~~~~~~~~~~~~~
+
+We don't support Windows for local |service| deployments during this 
+Public Preview stage.
+
+Local Machine Issues
+~~~~~~~~~~~~~~~~~~~~
+
+If the {+atlas-cli+} slows down to an unusable state after you create 
+multiple local |service| deployments and load the sample data, you 
+might have reached the limits of your machine.
+
+Consider allocating more memory:
+
+1. Stop the Podman machine:
+
+   .. code-block:: sh
+
+      podman machine Stop
+
+#. Allocate more memory:
+
+   .. code-block:: sh
+
+      podman machine set --cpus 2 --memory 2048
```

Comment:
```suggestion
      podman machine set --cpus 3 --memory 5120
```

Reply:
podman machine set --cpus 2 --memory 2048 is what we set by default, so it's ideal to add more capacity tan this